src/main/java/org/isf/exam/rest/ExamController.java

```diff
@@ -43,108 +47,120 @@
 import org.springframework.web.bind.annotation.PostMapping;
 import org.springframework.web.bind.annotation.PutMapping;
 import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.ResponseStatus;
 import org.springframework.web.bind.annotation.RestController;
 
 import io.swagger.v3.oas.annotations.security.SecurityRequirement;
 import io.swagger.v3.oas.annotations.tags.Tag;
 
-@RestController(value = "/exams")
 @Tag(name = "Exams")
+@RestController(value = "/exams")
```

Comment:
We are not going to specify "value" property on @RestController annotation

```suggestion
@RestController
```

Reply:
See OH2-403